.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+	"printWidth": 100,
+	"singleQuote": true,
+	"trailingComma": "all",
+	"useTabs": true,
+	"arrowParens": "avoid"
+}
```

Comment:
Where did this come from? Do we use this same configuration in other projects? Is it compatible with the standard ChainSafe ESLint configuration?

Reply:
Its the same prettier config thats being used in web3.js 
https://github.com/web3/web3.js/blob/4.x/templates/.prettierrc.json.tmpl